lib/wlib/memory/Allocator.cpp

```diff
@@ -55,7 +52,7 @@ wlp::Allocator::Allocator(uint16_t blockSize, uint16_t poolSize, wlp::Allocator:
 
         // Fill m_pPool with m_poolSize blocks
         wlp::Allocator::Block *pBlock = m_pPool;
-        for (size_type i = 1; i < m_poolTotalBlockCnt; i++) {
+        for (uint16_t i = 1; i < m_poolTotalBlockCnt; i++) {

```

Comment:
What's the reason for ditching `size_type`?

Reply:
I did not wanna include `Types.h` in any of these memory classes lol idk

Reply:
I can actually add size_type back because it is a `cpp` file. I just did not want a class including this to have `String`

Reply:
we can probably move the string typedefs to a `String.h` file

Reply:
yeah I have separated them now